app/controllers/dor_controller.rb

```diff
@@ -15,15 +15,19 @@ def reindex
   end
 
   def reindex_from_cocina
-    cocina_with_metadata = build_model_and_metadata(cocina_json: params[:cocina_object].presence,
+    # params[:cocina_object] is itself an ActionController::Parameters instance, hence the #require and #to_unsafe_h
```

Comment:
I think this is fine, but I also think that it is well known (and well documented within Rails) that params in a Rails controller returns AC::Parameters

Reply:
ah yeah, i wasn't totally sure about putting that comment in, but i got tripped up for a bit by `params[:cocina_object]` returning a `Parameters` instance while the two date values just returned strings, so i thought it might be helpful when coming back to the code.